components/commands/common/accelerator_pref_manager.cc

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/commands/common/accelerator_pref_manager.h"
+
+#include <vector>
+
+#include "base/containers/flat_map.h"
+#include "base/strings/string_number_conversions.h"
+#include "base/values.h"
+#include "brave/components/commands/common/accelerator_parsing.h"
+#include "components/prefs/pref_service.h"
+#include "components/prefs/scoped_user_pref_update.h"
+#include "ui/base/accelerators/accelerator.h"
+
+namespace commands {
+namespace {
+// A dictionary of command_id: [ "shortcut" ]
+// for example:
+// { 1: [ "Control+KeyC", "Control+KeySpace" ] }
+constexpr char kAcceleratorsPrefs[] = "brave.accelerators";
+}  // namespace
+
+// static
+void AcceleratorPrefManager::RegisterProfilePrefs(
+    PrefRegistrySimple* registry) {
+  registry->RegisterDictionaryPref(kAcceleratorsPrefs);
+}
+
+AcceleratorPrefManager::AcceleratorPrefManager(PrefService* prefs)
+    : prefs_(prefs) {}
+AcceleratorPrefManager::~AcceleratorPrefManager() = default;
```

Comment:
super nit: empty line

```suggestion

AcceleratorPrefManager::~AcceleratorPrefManager() = default;
```